File: Python/ex01/prime5to50.py
# Check if former devided by latter
def is_former_divisor_of_latter(n, m):
    return (n % m == 0)

# Check if it is prime number, only works in [5,50]
def is_prime(n):
    cnt = 0
    for i in range(2, n):
        if (is_former_divisor_of_latter(n, i)):
            cnt += 1
    return (cnt == 0)

# from 5 to 50
for i in range(5, 51):
    if is_prime(i):
        print('%d is a prime number' %i)

File: Web/ex01/prime5to50.js
function isPrime(n) {
    for (let i = 2; i < n; i++) {
        if (n % i === 0) return false;
    }
    return true;
}

for (let i = 5; i <= 50; i++) {
    if (isPrime(i)) {
        document.write(`${i} is a prime number`);
        document.write("<br/>");
    }
}
